frb_codegen/src/generator/dart/mod.rs

```diff
@@ -297,11 +297,10 @@ fn generate_dart_implementation_code(
 
 fn generate_file_prelude() -> DartBasicCode {
     DartBasicCode {
-        import: format!("{}
+        import: format!("{CODE_HEADER} @ {CARGO_PKG_VERSION}
```

Comment:
uber-nit: What about we put pkg version *into* the CODE_HEADER variable, otherwise we are having something like `Generated by `flutter_rust_bridge`. @ 1.42.0` which has a period there

Reply:
done in 9da58a7be169b7d669486b892d4a4d472324264a